diff --git a/benchmarks/run_kurobako.py b/benchmarks/run_kurobako.py
index 40f02b6de..46bb9f857 100644
--- a/benchmarks/run_kurobako.py
+++ b/benchmarks/run_kurobako.py
@@ -106,11 +106,11 @@ if __name__ == "__main__":
     parser.add_argument("--n-runs", type=int, default=100)
     parser.add_argument("--n-jobs", type=int, default=10)
     parser.add_argument("--n-concurrency", type=int, default=1)
-    parser.add_argument("--sampler-list", type=str, default="RandomSampler TPESampler")
+    parser.add_argument("--sampler-list", type=str, default="CmaEsSampler CmaEsSampler")
     parser.add_argument(
         "--sampler-kwargs-list",
         type=str,
-        default=r"{} {\"multivariate\":true\,\"constant_liar\":true}",
+        default=r"{} {\"with_margin\":true\}",
     )
     parser.add_argument("--pruner-list", type=str, default="NopPruner")
     parser.add_argument("--pruner-kwargs-list", type=str, default="{}")
diff --git a/setup.py b/setup.py
index e9fd69755..c1b807321 100644
--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,14 @@ def get_install_requires() -> List[str]:
     # please update `test-with-lower` in `.github/workflows/tests.yml` as well.
     requirements = [
         "alembic>=1.5.0",
-        "cmaes>=0.9.0",
+        # "cmaes>=0.9.0",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@c70716aa34a2d297e61f00e0bc0ce2cb8b99824b",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@919e4ddd06006fd4447890faaa717b935987eb6d",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@4034f4977a5fabda48f992638e439e7cbaf82747",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@90435ba1cab15701633a031a94bfaa19b5fb5ddb",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@25370a5228aaf51780118f3bbcca412864bedd8b",
+        # "cmaes@git+https://github.com/knshnb/cmaes.git@484e2d597275cfa81a525692514e3bb15a8f20c2",
+        "cmaes@git+https://github.com/knshnb/cmaes.git@c8b39aba37ee4bddf7f04aa691a2e61ce74a1bcb",
         "colorlog",
         "numpy",
         "packaging>=20.0",
