diff --git a/benchmarks/run_kurobako.py b/benchmarks/run_kurobako.py
index 40f02b6de..0ac9a5a24 100644
--- a/benchmarks/run_kurobako.py
+++ b/benchmarks/run_kurobako.py
@@ -102,11 +102,11 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--path-to-kurobako", type=str, default="")
     parser.add_argument("--name-prefix", type=str, default="")
-    parser.add_argument("--budget", type=int, default=80)
+    parser.add_argument("--budget", type=int, default=200)
     parser.add_argument("--n-runs", type=int, default=100)
-    parser.add_argument("--n-jobs", type=int, default=10)
+    parser.add_argument("--n-jobs", type=int, default=16)
     parser.add_argument("--n-concurrency", type=int, default=1)
-    parser.add_argument("--sampler-list", type=str, default="RandomSampler TPESampler")
+    parser.add_argument("--sampler-list", type=str, default="TPESampler TPESampler")
     parser.add_argument(
         "--sampler-kwargs-list",
         type=str,
