diff --git a/benchmarks/run_kurobako.py b/benchmarks/run_kurobako.py
index 14edaf427..8953d7ad7 100644
--- a/benchmarks/run_kurobako.py
+++ b/benchmarks/run_kurobako.py
@@ -55,9 +55,9 @@ def run(args: argparse.Namespace) -> None:
             f"pruner_list: {pruner_list}, pruner_keyword_arguments: {pruner_kwargs_list}."
         )
 
-    for sampler, sampler_kwargs in zip(sampler_list, sampler_kwargs_list):
+    for i, (sampler, sampler_kwargs) in enumerate(zip(sampler_list, sampler_kwargs_list)):
         for pruner, pruner_kwargs in zip(pruner_list, pruner_kwargs_list):
-            name = f"{args.name_prefix}_{sampler}_{pruner}"
+            name = f"{args.name_prefix}_{sampler}_{pruner}_{i}"
             cmd = (
                 f"{kurobako_cmd} solver --name {name} optuna --loglevel debug "
                 f"--sampler {sampler} --sampler-kwargs {sampler_kwargs} "
@@ -87,7 +87,7 @@ def run(args: argparse.Namespace) -> None:
     subprocess.run(cmd, shell=True)
 
     cmd = (
-        f"cat {result_filename} | {kurobako_cmd} plot curve --errorbar -o {args.out_dir} --xmin 10"
+        f"cat {result_filename} | {kurobako_cmd} plot curve --errorbar -o {args.out_dir} --xmin 50"
     )
     subprocess.run(cmd, shell=True)
 
@@ -96,16 +96,28 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--path-to-kurobako", type=str, default="")
     parser.add_argument("--name-prefix", type=str, default="")
-    parser.add_argument("--budget", type=int, default=80)
+    parser.add_argument("--budget", type=int, default=2000)
     parser.add_argument("--n-runs", type=int, default=100)
-    parser.add_argument("--n-jobs", type=int, default=10)
+    parser.add_argument("--n-jobs", type=int, default=8)
     parser.add_argument("--n-concurrency", type=int, default=1)
-    parser.add_argument("--sampler-list", type=str, default="RandomSampler TPESampler")
+    parser.add_argument(
+        "--sampler-list",
+        type=str,
+        default="QMCSampler RandomSampler TPESampler CmaEsSampler CmaEsSampler",
+    )
     parser.add_argument(
         "--sampler-kwargs-list",
         type=str,
-        default=r"{} {\"multivariate\":true\,\"constant_liar\":true}",
+        default=r"{} {} {\"multivariate\":true} {} {\"use_cma_with_margin\":true}",
     )
+    # parser.add_argument(
+    #     "--sampler-list", type=str, default="CmaEsSampler CmaEsSampler"
+    # )
+    # parser.add_argument(
+    #     "--sampler-kwargs-list",
+    #     type=str,
+    #     default=r"{} {\"use_cma_with_margin\":true}",
+    # )
     parser.add_argument("--pruner-list", type=str, default="NopPruner")
     parser.add_argument("--pruner-kwargs-list", type=str, default="{}")
     parser.add_argument("--seed", type=int, default=0)
diff --git a/docs/Makefile b/docs/Makefile
index 1c59a8be3..a53e0d71b 100644
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W --keep-going
+SPHINXOPTS    = -W --keep-going --nitpicky
 SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = Optuna
 SOURCEDIR     = source
